integration-test-jdbc/src/test/kotlin/integration/jdbc/JdbcUpdateBatchTest.kt

```diff
@@ -192,4 +195,15 @@ class JdbcUpdateBatchTest(private val db: JdbcDatabase) {
             assertFalse(a.location.endsWith("]"))
         }
     }
+
+    @Test
+    fun idColumnOnlyEntity() {
+        val a = Meta.idColumnOnlyAddress
+        val query = QueryDsl.from(a).limit(1)
+        val address = db.runQuery { query.single() }
+        assertThrows<IllegalArgumentException> {
+            val updateQuery = QueryDsl.update(a).single(address)
```

Comment:
Since this file is JdbcUpdateBatchTest.kt, I think it would be appropriate to use the batch function here instead of the single function.

Reply:
Sorry, I fixed.